src/components/Atoms/MarqueeAtom/MarqueeAtom.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import type { FC, ReactNode } from "react";
+// import styles from "./MarqueeAtom.module.css"; // Import CSS module
+
+type Props = {
+  children: ReactNode;
+};
+
+const MarqueeAtom: FC<Props> = ({ children }) => {
```

Comment:
Do not use FC for type alias
change to `const MarqueeAtom = (props: Props) => {`